### ISSUE TITLE
Duplicate Data in response

### ISSUE THREAD

**Issue by spaquis** [nutboltu/storybook-addon-mock#131]
Hey,
Thanks for your plugin, In last version 3.0.0, the data attribute appear twice  in the response when using it with axios.

<img width="1094" alt="image" src="https://user-images.githubusercontent.com/27917715/193470227-a6a9d3c3-59a0-43eb-a321-b717d140bb0b.png">

```
parameters: {
    mockData: [
      {
        url: 'https://XXX/api/XXX',
        method: 'GET',
        status: 200,
        response: { data: [mockData] },
      },
    ],
  },
  ```
 

Thanks for help.

**Comment by nutboltu** [comment]
@spaquis It looks good. You pass data inside the response. `response` is a JSON object expected to be returned as data in Axios. So the second `data` object is the object you passed insiide the response.

You can either pass like below

```
parameters: {
    mockData: [
      {
        url: 'https://XXX/api/XXX',
        method: 'GET',
        status: 200,
        response: [mockData],
      },
    ],
  },
```

**Comment by spaquis** [comment]
Hi @nutboltu ,
Thanks for your reply but i already tried to put [mockData] directly in response but if i do that the mock is not recognized anymore, in network i see the original request.
I make the same change in other request when the data in not an array and it's work so i think the problem appear only when the response is an array.

**Comment by nutboltu** [comment]
@spaquis This is fixed in [3.2.0](https://github.com/nutboltu/storybook-addon-mock/releases/tag/3.2.0). So now it also accepts an array as a response type.
